serve/graph/schema/filter/transaction_filter.graphql

```diff
@@ -65,6 +65,54 @@ input TransactionFilter {
   `success` enables you to filter between successful and unsuccessful transactions.
   """
   success: Boolean
+
+  """
+  `events` are the events that the transaction has emitted.
+  `events` filters transactions with specific events.
+  """
+  events: [TransactionEventInput!]
+}
+
+"""
+Transaction's event to filter transactions.
+"TransactionEventInput" can be configured as a filter with a transaction event's `type` and `pkg_path` and `func`, and `attrs`.
+"""
+input TransactionEventInput {
+  """
+  `type` is the type of transaction event emitted.
+  """
+  type: String
+
+  """
+  `pkg_path` is the path to the package that emitted the event.
+  """
+  pkg_path: String
+
+  """
+  `func` is the name of the function that emitted the event.
+  """
+  func: String
+
+  """
+  `attrs` filters transactions whose events contain attributes.
+  """
+  attrs: [TransactionEventAttributeInput!]
```

Comment:
Please, specify how this filter works. Are these attributes inclusive or exclusive (`attr1 && attr2`, or `attr1 || attr2` ?)?

Reply:
Thank you for pointing out the shortfall.
Filters in the form of an array work exclusively. (`attr1 || attr2`)

I'm going to add a comment.